wit/wavs_worker.wit

```diff
@@ -107,6 +119,11 @@ interface layer-types {
     block-height: u64
   }
 
+  record trigger-data-cron {
+    trigger-time: timestamp,
```

Comment:
what is the difference between these two?

Reply:
It seems to me we can remove `execution_time`, please see https://github.com/Lay3rLabs/WAVS/pull/478#discussion_r2024137901